packages/@react-spectrum/numberfield/src/NumberField.tsx

```diff
@@ -63,13 +65,16 @@ function NumberField(props: SpectrumNumberFieldProps, ref: FocusableRef<HTMLElem
     {
       'spectrum-InputGroup--quiet': isQuiet,
       'is-invalid': props.validationState === 'invalid',
```

Comment:
Think this needs to be `'spectrum-InputGroup--invalid': validationState === 'invalid'`, noticed the following when on mobile and focus is within an invalid numberfield:
![image](https://user-images.githubusercontent.com/9661127/107589658-57248680-6bbb-11eb-8b8d-e10e7e69f3ef.png)

Reply:
same for non mobile: 
![image](https://user-images.githubusercontent.com/9661127/107589703-6efc0a80-6bbb-11eb-9063-5b455e5dabc8.png)